modules/core/05-port/types/errors.go

```diff
@@ -6,8 +6,9 @@ import (
 
 // IBC port sentinel errors
 var (
-	ErrPortExists   = sdkerrors.Register(SubModuleName, 2, "port is already binded")
-	ErrPortNotFound = sdkerrors.Register(SubModuleName, 3, "port not found")
-	ErrInvalidPort  = sdkerrors.Register(SubModuleName, 4, "invalid port")
-	ErrInvalidRoute = sdkerrors.Register(SubModuleName, 5, "route not found")
+	ErrPortExists     = sdkerrors.Register(SubModuleName, 2, "port is already binded")
+	ErrPortNotFound   = sdkerrors.Register(SubModuleName, 3, "port not found")
+	ErrInvalidPort    = sdkerrors.Register(SubModuleName, 4, "invalid port")
+	ErrInvalidRoute   = sdkerrors.Register(SubModuleName, 5, "route not found")
+	ErrInvalidVersion = sdkerrors.Register(SubModuleName, 6, "invalid port version")
```

Comment:
What is this error used for? There is no port version

Reply:
Ah yes! Good spot, added by mistake and didn't end up using this.
Later realised it should be scoped to the application module and `ErrInvalidVersion` already existed for `transfer` 👍